autorandr.py

```diff
@@ -801,11 +801,23 @@ def main(argv):
         if options["--remove"] not in profiles.keys():
             raise AutorandrException("Cannot remove profile '%s':\nThis profile does not exist." % options["--remove"])
         try:
+            remove = True
             profile_folder = os.path.join(profile_path, options["--remove"])
-            shutil.rmtree(profile_folder)
+            profile_dirlist = os.listdir(profile_folder)
+            profile_dirlist.remove("config")
+            profile_dirlist.remove("setup")
+            if profile_dirlist:
+                print("Profile folder '%s' contains the following:\n---\n%s\n---" % (options["--remove"], "\n".join(profile_dirlist)))
+                response = input("Do you really want to remove profile '%s'? If so, type 'yes': " % options["--remove"]).strip()
```

Comment:
I must correct myself, though [this site suggests it would work](http://python-future.org/compatible_idioms.html#raw-input), the import does not work in Python 2. [This solution](http://python3porting.com/differences.html#input-and-raw-input) works:

    try:
        input = raw_input
    except NameError:
        pass

This should go directly below the `import` lines.